servobuild.example

```diff
@@ -40,8 +40,10 @@ debug-mozjs = false
 # Any optional flags that will be added to $RUSTFLAGS
 #rustflags = ""
 
-# Enable rustc’s incremental compilation
+# Enable or disable rustc’s incremental compilation
+# The default is enabled in debug mode only.
 #incremental = false
+#incremental = true
```

Comment:
Having both is a bit weird.

Reply:
I suppose it’s not clear enough, but the intent is to show that leaving it out of the config file is a third behavior different from `true` or `false`. (It’s `None` in Python. Does TOML have explicit `null` values? I don’t find it in https://github.com/toml-lang/toml.)

Another option might be to make the `"debug"` string that third value, or something.